hudi-common/src/main/java/org/apache/hudi/common/model/debezium/PostgresDebeziumAvroPayload.java

```diff
@@ -71,6 +72,19 @@ protected boolean shouldPickCurrentRecord(IndexedRecord currentRecord, IndexedRe
     return insertSourceLSN < currentSourceLSN;
   }
 
+  @Override
+  public Option<IndexedRecord> combineAndGetUpdateValue(IndexedRecord currentValue, Schema schema, Properties properties) throws IOException {
+    // Specific to Postgres: If the updated record has TOASTED columns,
+    // we will need to keep the previous value for those columns
```

Comment:
check lines 89 to 99. adding this method which takes in properties and it calls respective super.combineAndGetUpdateValue(currentValue, schema, properties) instead of combineAndGetUpdateValue(currentValue, schema) as you see in line 89 to 99. 
There is some duplication. but few couple of lines, did not feel we should introduce a private method. let me know wdyt.

Reply:
That's ok. More than duplication of code, I was concerned about the need of ding it here. Makes sense to me now.